docs/whats-new/whats-new.md

```diff
@@ -5,6 +5,7 @@ This page summarises the new features, improvements and bug fixes completed in e
 ## Latest changes
 
 **Production release notes**
+- [Production release 31 August 2022](whats-new/staging-release-notes?id=production-release-31-august-2022)
```

Comment:
these 2 lines should point to **whats-new/production-release-notes**

Reply:
I think i have updated.. one link was pointing to staging in the prod release notes.